inc/config/query-runner/query-runner.php

```diff
@@ -126,6 +127,12 @@ private function get_field_value( array|string $field_value, string $default_val
 			: ( $field_value[0] ?? $default_value );
 
 		switch ( $field_type ) {
+			case 'base64':
+				return base64_decode( $field_value_single );
+
+			case 'html':
+				return $field_value_single;
```

Comment:
I confirmed, as expected,  core escaping happens further on in execution--somewhere closer to output.

<img width="295" alt="Screenshot 2024-09-06 at 2 21 14 PM" src="https://github.com/user-attachments/assets/4b7714dc-f223-461b-bfd7-69416d37628a">

<img width="426" alt="Screenshot 2024-09-06 at 2 21 01 PM" src="https://github.com/user-attachments/assets/0e8adc6d-563e-438f-aea8-be9b2583fcca">

Reply:
escapin late :)